### DISCUSSION TITLE
How do I deselect a clock after updating the Clock Tree Viewer

### DISCUSSION THREAD

**Discussion by oharboe** [The-OpenROAD-Project/OpenROAD#4624] [General]
Once I have updated the clock tree viewer, how do I deselect the clock tree(orange and red) that is now hiding everything in the chip view?

![image](https://github.com/The-OpenROAD-Project/OpenROAD/assets/2798822/3abf5fae-24ca-4a84-b00d-d7d30d74ab5a)

**Comment by maliberty** [comment]
Right click in the widget and you can turn off the display.

**Reply by oharboe** [reply]
![Screenshot_2024-02-06_07-59-44](https://github.com/The-OpenROAD-Project/OpenROAD/assets/2798822/65f72b40-e61a-44a6-9fda-8ef15d971985)
